benchmark-compare.js

```diff
@@ -54,19 +54,24 @@ if (!choices.length) {
     table.push([':--', '--:', ':-:', '--:', '--:'])
   }
 
-  choices.forEach((result) => {
-    let data = readFileSync(`${resultsPath}/${result}.json`)
-    data = JSON.parse(data.toString())
-    const beBold = result === 'fastify'
-    const { hasRouter = false } = info(result) || {}
-    table.push([
-      bold(beBold, chalk.blue(result)),
-      bold(beBold, hasRouter ? '✓' : '✗'),
-      bold(beBold, (data.requests.average).toFixed(1)),
-      bold(beBold, (data.latency.average).toFixed(2)),
-      bold(beBold, (data.throughput.average / 1024 / 1024).toFixed(2))
-    ])
+  choices.map(file => {
+    let content = readFileSync(`${resultsPath}/${file}.json`)
+    return JSON.parse(content.toString())
   })
+    .sort((a, b) => {
+      return parseFloat(b.requests.mean) - parseFloat(a.requests.mean)
+    })
+    .forEach((data) => {
+      const beBold = data.server === 'fastify'
+      const { hasRouter = false } = info(data.server) || {}
+      table.push([
```

Comment:
You could return early if `data.requests.average` is 0, to ignore broken results like `take-five`

Reply:
Good catch!
Updated

I didn't remove the framework from the list, but I will show the `N/A` - not apply string